src/pages/MainPage/MainPage.jsx

```diff
@@ -33,18 +45,40 @@ const MainPage = () => {
       setShowSkeleton(false);
     };
   }, [dispatch]);
-
   const addToCart = (product, i) => {
     const currentProduct = {
       product,
-      data: Date.now(),
     };
+
     dispatch(addUser(currentProduct));
   };
 
+  const removeFromFavoritess = (id) => {
+    dispatch(removeFromFavorites(id));
+  };
+
   function showBasket(cond) {
     setShow(cond);
   }
+  const FilterData = (event) => {
```

Comment:
FilterData shoud start from `lowerCase`

Reply:
and create this search function outside a componenet and use filter on your users list